diff --git a/CMakeLists.txt b/CMakeLists.txt
index 947341d..e3e3c83 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,9 +7,7 @@ cmake_policy(SET CMP0068 NEW) # new in 3.9. The NEW behavior of this policy is t
 project(QtNodesLibrary CXX)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
-set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
-set(CMAKE_DISABLE_SOURCE_CHANGES  ON)
 set(OpenGL_GL_PREFERENCE LEGACY)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
